specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-01-01-preview/Metadata.json

```diff
@@ -628,6 +636,63 @@
       "type": "string",
       "format": "date"
     },
+    "metadataCustomVersion": {
+      "description": "The custom version of the content. A optional free text",
+      "type": "string"
+    },
+    "metadataContentSchemaVersion": {
+      "description": "Schema version of the content. Can be used to distinguish between different flow based on the schema version",
+      "type": "string"
+    },
+    "metadataIcon": {
```

Comment:
>metadataIcon [](http://example.com/codeflow?start=5&length=12)

Is this a known identifier? (e.g. a Guid?) might be good to specify the id format (in case there's one)

Reply:
no, just a name, can be "bob", "roey", "random_name2"